rosetta-lang/src/main/java/com/regnosys/rosetta/formatting2/RosettaFormatter.xtend

```diff
@@ -236,6 +237,18 @@ class RosettaFormatter extends AbstractRosettaFormatter2 {
 		ele.regionFor.assignment(nameAssignment_1)
 			.prepend[oneSpace]
 	}
+	
+	def dispatch void format(Choice ele, extension IFormattableDocument document) {
+		ele.regionFor.keyword('choice')
+			.append[oneSpace]
+		ele.regionFor.keyword(':')
+			.prepend[noSpace]
```

Comment:
Similar to `Data`, a `Choice` can also have a definition (documentation), so it would be good to add `ele.formatDefinition(document)` too.

Reply:
That'll make sure that `<"my docs">` are formatted correctly. I.e., not as `  <    "my docs"  >  `